avocado/utils/disk.py

```diff
@@ -9,18 +9,84 @@
 #
 # See LICENSE for more details.
 #
-# This code was inspired in the autotest project,
+# Copyright: 2017 IBM
+# Author: Narasimhan V sim@linux.vnet.ibm.com
 #
+# freespace() function was inspired in the autotest project,
 # client/base_utils.py
 
 
 """
 Disk utilities
 """
 
+import logging
 import os
+from . import process
+
+
+LOGGER = logging.getLogger('avocado.test')
+
+
+class FSException(Exception):
+    """
+    Base Exception Class for all exceptions
```

Comment:
As you have two of `for all exceptions` it implies it's not really for all exceptions. I assume you want to use one for `all filesystem-related exceptions` and the other for `disk-related exceptions`, right?